demo/src/app/components/components.e2e-spec.ts

```diff
@@ -10,60 +9,57 @@ const COMPONENTS = [
   'popover', 'progressbar', 'rating', 'table', 'tabset', 'timepicker', 'toast', 'tooltip', 'typeahead'
 ];
 
-describe(`Components`, () => {
+test.describe(`Components`, () => {
 
   let messages: ConsoleMessage[] = [];
-  let page: Page;
 
-  beforeAll(async () => {
-    page = await test.newPage(`${BASE_URL}/components`);
+  test.beforeEach(async({page, baseURL}) => {
+    messages = [];
+    await page.goto(`${baseURL}/components`);
     page.on('console', msg => messages.push(msg));
 
     await page.waitForSelector(SELECTOR_SIDE_NAV_COMPONENT_LINKS);
   });
 
-  afterAll(async() => {
-    await test.destroy();
-  }, 120000);
-
-  it('should cover all components we have', async() => {
-    for (const link of await page.$$(SELECTOR_SIDE_NAV_COMPONENT_LINKS)) {
-      const componentName = await link.innerText();
-      expect(COMPONENTS).toContain(componentName.toLowerCase(), `'${componentName}' is not covered by demo e2e tests`);
+  test.afterEach(async() => {
+    if (messages.length > 0) {
+      console.log(messages);
+      test.fail(true, 'Unexpected console messages found');
     }
   });
 
-  COMPONENTS.forEach(component => {
-    describe(`${component} page`, () => {
+  test('should work', async({page}) => {
```

Comment:
should work? :)

Reply:
I have replaced it with "should load successfully". Is it now precise enough?